backend/backend/settings.py

```diff
@@ -0,0 +1,4 @@
+from google.oauth2 import service_account
+
+client_file = 'sa-mock-ai-interviewer'
+credentials = service_account.Credentials.from_service_account_file(client_file + '.json')
```

Comment:
This file and the variable names are abit generic
E.g. importing this credentials variable would be done by using `from backend.settings import credentials`, This doesn't convey alot of info
There's already a `conf.py` file i've added to base package level. If you could add a function in there called `get_google_service_account_credentials()` or something along these lines then it would be abit clearer